src/plugins/d3/renderer/hooks/useSeries/prepare-line-series.ts

```diff
@@ -103,6 +113,8 @@ export function prepareLineSeries(args: PrepareLineSeriesArgs) {
                 allowOverlap: get(series, 'dataLabels.allowOverlap', false),
             },
             marker: prepareMarker(series, seriesOptions),
+            dashStyle: prepareDashStyle(series),
+            linecap: prepareLinecap(series),
```

Comment:
Below the property is added to the types for seriesOptions
Is it not being used?